python/doc/theory/reliability_sensitivity/taylor_moments.rst

```diff
@@ -1,144 +1,106 @@
-Taylor variance decomposition
------------------------------
-
-| The Taylor variance decomposition (also referred as quadratic cumul method
-  or perturbation method) is a probabilistic approach designed to propagate
-  the uncertainties of the input variables :math:`\uX` through the model
-  :math:`h` towards the output variables :math:`\uY`. It enables to
-  access the central dispersion (expectation and variance) of the output
-  variables.
-| This method is based on a Taylor decomposition of the output variable
-  :math:`\uY` towards the :math:`\uX` random vectors around the mean
-  point :math:`\muX`. Depending on the order of the Taylor decomposition
-  (classically first order or second order), one can obtain different
-  formulas. For easiness of the reading, we first present the formulas
-  with :math:`n_Y=1` before the ones obtained for :math:`n_Y > 1`.
-| As :math:`Y=h(\uX)`, the Taylor decomposition around
-  :math:`\ux = \muX` at the second order yields to:
-
-| **Case** :math:`n_Y=1`
+.. _taylor_expansion_moments:
 
-.. math::
+Taylor expansion moments
+------------------------
 
-    Y = h(\muX) + < \underline{\nabla} h(\muX) , \: \uX - \muX> + \frac{1}{2}<<\underline{\underline{\nabla }}^2 h(\muX,\: \underline{\mu}_{\:X}),\: \uX - \muX>,\: \uX - \muX> + o(\Cov \uX)
+One way to evaluate the central dispersion (expectation and variance) of the variable :math:`\uY=h(\uX)`
+is to use the Taylor expansion of the function :math:`h: \Rset^d \rightarrow \Rset^q` at
+the mean point :math:`\vect{\mu} = \Expect{\uX}`.
 
```

Comment:
This will remove the unnecessary newline:
![image](https://github.com/openturns/openturns/assets/31351465/8e133c45-f732-4045-a412-22738b5337bd)


```suggestion
```

Reply:
ok